### PR TITLE
feat: create `CustomerGroup` model

### PR DESCRIPTION
**Pull request by stephsprinkle**
#### Summary
- Created `CustomerGroup` and draft model
- Fixed typo in `Zone` readme

### CODE REVIEW

File: models/customer-group/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# @commercetools-test-data/zone
```
Comment (tdeekens) [inline models/customer-group/CHANGELOG.md:1]: Should be empty and `customer-group` I think

Comment (stephsprinkle) [reply models/customer-group/CHANGELOG.md:1]: Copy pasta strikes again 🙃

**Review by tdeekens — APPROVED** [review]

File: models/zone/README.md [hunk 2]
```diff
@@ -17,5 +17,5 @@ import type { TZone, TZoneDraft } from '@commercetools-test-data/zone';
 import * as Zone from '@commercetools-test-data/zone';
 
 const zone = Zone.random().build<TZone>();
-const zoneDraft = Zone.Zone.random().build<TZoneDraft>();
+const zoneDraft = Zone.ZoneDraft.random().build<TZoneDraft>();
```
Comment (tylermorrisford) [inline models/zone/README.md:20]: 🙏 thank you 🙏

File: models/customer-group/src/generator.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TCustomerGroup>({
+  fields: {
+    id: fake((f) => f.datatype.uuid()),
+    version: sequence(),
+    key: fake((f) => f.lorem.slug(2)),
+    name: fake((f) => f.lorem.words(2)),
+    custom: [],
```
Comment (jaikamat) [inline models/customer-group/src/generator.ts:16]: nit: I don't think `CustomFields` is an array, perhaps `null` here might work?

File: models/customer-group/src/customer-group-draft/generator.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TCustomerGroupDraft } from '..//types';
+
+// https://docs.commercetools.com/api/projects/customerGroups#customergroupdraft
+
+const generator = Generator<TCustomerGroupDraft>({
+  fields: {
+    groupName: fake((f) => f.lorem.words(2)),
+    key: fake((f) => f.lorem.slug(2)),
+    custom: [],
```
Comment (jaikamat) [inline models/customer-group/src/customer-group-draft/generator.ts:10]: nit: same observation here 🤷‍♂️

**Review by jaikamat — APPROVED** [review]
🏅

File: models/customer-group/src/generator.ts (hunk 3 above)
Comment (stephsprinkle) [reply models/customer-group/src/generator.ts:16]: Wow you are 100% correct, I'll update all of these to null

**Review by emmenko — COMMENTED** [review]
Besides what @jaikamat noticed, looks good 👌

**Review by tylermorrisford — APPROVED** [review]
#rocketShip

**Pull request merged**